packages/ts/file-router/src/vite-plugin/collectRoutesFromFS.ts

```diff
@@ -33,10 +33,11 @@ export type CollectRoutesOptions = Readonly<{
 async function checkFile(url: URL | undefined, logger: Logger): Promise<URL | undefined> {
   if (url) {
     const contents = await readFile(url, 'utf-8');
-    if (contents === '') {
+    if (contents === '' || contents.trim() === '') {
```

Comment:
Using `trim` is enough here
```suggestion
    if (contents.trim() === '') {
```

Reply:
Done.